src/contexts/maze/GameContextProvider.tsx

```diff
@@ -493,7 +495,7 @@ export const GameContextProvider = ({ children }: props) => {
     gameOverRefSent.current = false;
 
     // Regenerate maze data
-    const rng = new RNG(newSeedIdResponse.seedId);
```

Comment:
It's important to keep using the newSeedIdResponse.seedId . Not using it makes it imposible to control that the match was not hacked for the backend.

Reply:
the reason is seedId increase by +1, so when user play new game the game maze looks pretty similar since the RNG is pretty close to last one, but we already store the data using the seedId, so why does it matter?

Reply:
Hmmm, my explanation was not correct.

The problem is that having the seedId you already know what is in each tile. A hacker could find which ones give him the most benefits and hack to use them.

We want the back to give it so that no one can have the power to hack using this method. It is also worth saying that the back stores which is the seedId that it delivered and which is the one that it received while the game ends.

We have to find another way to increase the randomness.